common/lib/dependabot/clients/bitbucket.rb

```diff
@@ -207,7 +207,7 @@ def compare(repo, previous_tag, new_tag)
 
       def get(url)
         response = Excon.get(
-          url,
+          URI::Parser.new.escape(url),
```

Comment:
Our linter suggested

```suggestion
          URI::DEFAULT_PARSER.escape(url),
```